README.md

```diff
@@ -2,9 +2,32 @@
 
 OpenTelemetry community content.
```

Comment:
small logo here? and link to website?

Reply:
also maybe make  "OpenTelemetry community content." an H1 instead of the word "Community"?